R/epinow.R

```diff
@@ -39,20 +39,11 @@
 #' # set number of cores to use
 #' old_opts <- options()
 #' options(mc.cores = ifelse(interactive(), 4, 1))
-#' # construct example distributions
-#' generation_time <- get_generation_time(
-#'  disease = "SARS-CoV-2", source = "ganyani"
-#' )
-#' incubation_period <- get_incubation_period(
-#'  disease = "SARS-CoV-2", source = "lauer"
-#' )
-#' reporting_delay <- dist_spec(
-#'   mean = convert_to_logmean(2, 1),
-#'   mean_sd = 0.1,
-#'   sd = convert_to_logsd(2, 1),
-#'   sd_sd = 0.1,
-#'   max = 10
-#' )
+#'
+#' # get example delays
```

Comment:
I think we now need some signposting to dist_spec or people have no way to know how to specify there own distributions.

Reply:
I recognise this is in the documentation for the parameters but I think many users don't read this.